pom.xml

```diff
@@ -89,6 +94,12 @@
                     </execution>
 
                 </executions>
+                <configuration>
+                    <excludes>
+                        <exclude>**/Main*</exclude>
+                        <exclude>**/ParseToCSV*</exclude>
```

Comment:
I currently can't see a class with this name in your project

Reply:
sorry, inattentive, renamed it, and forgot to change